integration/test/dashboard_json_test.go

```diff
@@ -200,14 +202,14 @@ func (suite *DashboardJsonTestSuite) TestJsonExpression() {
 	}
 
 	if isRestFailed {
-		log.Warn().Msgf("Rest Test validation is failed. Pls check logs above. Total Missing Rest counters %d", len(perfErrorInfoList))
+		assert.Fail(suite.T(), "Rest Test validation is failed. Pls check logs above. Count of Missing Rest counters %d", len(perfErrorInfoList))
 	} else {
 		log.Info().Msg("Rest Validation looks good!!")
 	}
 
 	// Fail only for Zapi Collector
```

Comment:
comment should be removed or changed to something like
Fail if either Rest or Zapi collectors have failures